ngs_mapper/file_formats.py

```diff
@@ -0,0 +1,67 @@
+from ngs_mapper.reads import sffs_to_fastq
+import os, sys
+from functools import partial
+from glob import glob
+import gzip
+from Bio import SeqIO
+
+{
+    'sff' : 'sff',
+    'ab1' : 'abi',  # SeqIO.convert(x, 'abi', y, 'fastq')
+}
+
+drop_ext = lambda s: '.'.join(s.split('.')[:-1])
```

Comment:
Probably a good idea. I have a slight reservation to introducing anything new right now, but that reservation is unfounded and probably due to old age.

What do you think?